kuksa-client/kuksa_client/cli_backend/ws.py

```diff
@@ -297,8 +297,6 @@ async def connect(self, _=None):
         subprotocols = ["VISSv2"]
         if not self.insecure:
             context = ssl.create_default_context()
-            context.load_cert_chain(
-                certfile=self.certificate, keyfile=self.keyfile)
```

Comment:
If we do not want to remove support for client auth then this statement must be modified by checking that both are not null before calling it